spec/support/content_item_helper.rb

```diff
@@ -0,0 +1,64 @@
+module ContentItemHelpers
+
+  def stub_call_to_content_store(content_store_url, web_url)
+    stub_request(:get, content_store_url).
```

Comment:
This can be shortened to:

```ruby
stub_request(:get, content_store_url).to_return(body: content_item('my_url'))
```

The rest are defaults/not relevant for this test.